web/js/userTasksReport.js

```diff
@@ -161,7 +197,33 @@ Ext.onReady(function () {
         width: 350,
         renderTo: 'content',
         defaults: {width: 230},
-        items: [{
+        items: [
+            new Ext.form.ComboBox({
```

Comment:
An equivalent way to define items is creating an object which contains the correct `xtype`field, instead of creating instances with new. For example, you could have added an object with `xtype: 'combo'`. Take a look at the combo for customer, defined just some lines below.